core/runtime.rs

```diff
@@ -353,15 +357,9 @@ impl JsRuntime {
   ) -> Result<(), AnyError> {
     self.shared_init();
 
-    let state_rc = Self::state(self);
-    let state = state_rc.borrow();
-
-    let scope = &mut v8::HandleScope::with_context(
-      self.v8_isolate.as_mut().unwrap(),
-      state.global_context.as_ref().unwrap(),
-    );
+    let context = self.global_context();
```

Comment:
This calls are creating clones of `v8::Global` instead of using `Option<&v8::Global<>>`, but I haven't noticed any perf hit on benchmarks